packages/rescript-downshift/src/Downshift.res

```diff
@@ -0,0 +1,6 @@
+module Select = Downshift_Select
+
+module Spread = {
+  @react.component
+  let make = (~props, ~children) => React.cloneElement(children, props)
+}
```

Comment:
Mentioned this in the brightmark PR - we could probably extract Spread in it's own rescript-index package.